public/assets/index.js

```diff
@@ -120,21 +120,43 @@ function handleDeltaUpdate(message) {
     if (message["name"] in block || block.includes(message["name"])) {
         return;
     }
-
-    var text = `<strong>${message["name"]}</strong> has gone ${mapState(message["state"])}! (${message["section"]})`;
-
+    
+    const subredditElement = document.getElementById(message["name"]);
+    const subredditStatusElement = subredditElement.querySelector("p");
+
+    const mappedState = mapState(message["state"]);
+    const prevState = subredditStatusElement.innerHTML;
+    
+    // create status update string
+    var text = `<strong>${message["name"]}</strong> (${message["section"]})<br>`
+        + `${prevState} → <strong>${mappedState}</strong>`;
+    
+    switch (message["state") {
+        case "PRIVATE":
+            if (prevState != "restricted") text += "!";
```

Comment:
thanks for pointing that out.

just to clarify something i found, should [this](https://github.com/username-is-required/reddark-remix-fork/blob/8502ec4a52d23e01fcd12cc11c05035507c7920b/public/assets/index.js#L165) be `message[“previous_state”]` rather than `subreddit[“previous_state”]`? just asking as i can’t see `subreddit` defined within the function. if it should be, i’ll fix it in the pr.